connectors/vault/testutil/helper.go

```diff
@@ -0,0 +1,51 @@
+// Copyright 2020 IBM Corp.
+// SPDX-License-Identifier: Apache-2.0
+
+package testutil
```

Comment:
why we need helper file if it has only 2 short functions used in only 1 other file? such helper files usually serve several other files. Do you expect tests to grow to several other files?

Reply:
I think since we have followed this design in all connectors we can conform to this design for now. It is more for uniformity of design that this is being followed. Also it makes the connectors independent of each other and also self contained in my opinion.